aggregator/src/test/java/com/redhat/cloud/notifications/DailyEmailAggregationJobTest.java

```diff
@@ -79,7 +78,7 @@ void shouldProcessFourSubscriptions() {
         helpers.addEmailAggregation("tenant", "unknown-bundle", "policies", "somePolicyId", "someHostId");
         helpers.addEmailAggregation("tenant", "unknown-bundle", "unknown-application", "somePolicyId", "someHostId");
 
-        final List<AggregationCommand> emailAggregations = testee.processAggregateEmails(LocalDateTime.now(UTC));
+        final List<AggregationCommand> emailAggregations = testee.processAggregateEmails(LocalDateTime.now(UTC), new CollectorRegistry());
```

Comment:
It would be great if the tests checked the metrics.

Reply:
Added-